packages/shared/src/components/squads/SquadFeedHeading.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactElement, useContext, useMemo } from 'react';
+import { QueryKey } from 'react-query';
+import { Button } from '../buttons/Button';
+import { PinIcon } from '../icons';
+import { Squad } from '../../graphql/sources';
+import { ActiveFeedContext } from '../../contexts';
+import { useSquadActions } from '../../hooks';
+
+interface SquadFeedHeadingProps {
+  squad: Squad;
+  feedQueryKey: QueryKey;
```

Comment:
What do you need this one for here?

Reply:
@DragosIorgulescu Just this question remains I think.

Reply:
this is to pass it down when invalidating the feed cache, I pushed one more commit to fix an issue with the cache update which was not causing the UI to reflect the changes properly. Let me know if this is good to merge

Reply:
Ah but you can get it from the context no?
No need to pass it I think?

Reply:
ah, my bad, loading it from context is much better. Updated 🙏